Sources/HotSoda/AbilityProtectedMiddleware.swift

```diff
@@ -73,6 +73,10 @@ public extension Router {
     func protected<T>(_ subject: T.Type, for types: [AbilityType]) -> Router where T: AbilityProtected, T: Model, T: Parameter, T.ResolvedParameter == Future<T> {
         return grouped(AbilityProtectedMiddleware<T>(for: types))
     }
+
+    func protected<T>(_ subject: T.Type, for types: [AbilityType], configure: (Router) -> Void) where T: AbilityProtected, T: Model, T: Parameter, T.ResolvedParameter == Future<T> {
```

Comment:
This function does't have no return value, so `protect` is more appropriate  than `protected`.